src/native/corehost/apphost/static/singlefilehost_freebsdexports.src

```diff
@@ -9,3 +9,7 @@ g_dacTable
 
 ; Used by profilers
 MetaDataGetDispenser
+
+; FreeBSD needs to reexport these
```

Comment:
Same warning as above, but I "fixed" this one instead. We only need to special case FreeBSD, and not OSX.